### PR TITLE
fix setup

### PR DESCRIPTION
**Pull request by AurelienJaquier**
This fixes the bad version number that we get on pypi

### CODE REVIEW

**Comment by AurelienJaquier** [comment]
The README.rst was not passing in the setup.py's long description so I had to fix it

File: README.rst [hunk 1]
```diff
@@ -1,64 +1,26 @@
...
-		<td>Gitter</td>
-		<td>
-			<a href="https://gitter.im/bluebrain/currentscape">
-			<img src="https://badges.gitter.im/Join%20Chat.svg"
-		</a>
-		</td>
-	</tr>
-	</table>
+
++----------------+------------+
```
Comment (wvangeit) [inline README.rst:5]: Does this work in rst files?

Comment (wvangeit) [reply README.rst:5]: Does it work in the doc though? I remember Alexander put this complex thing to make things work in sphinx, but maybe that was not necessary.

Comment (AurelienJaquier) [reply README.rst:5]: I just checked, it works also in the doc. I noticed that we weren't using readme in the docs, so I added it, with a few more fixes.

**Review by wvangeit — APPROVED** [review]

**Pull request merged**